docs/website/docusaurus.config.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const theme = require('prism-react-renderer/themes/github');
+
+module.exports = {
+  title: 'Renovate Docs',
+  tagline: 'Documentation for Renovate',
+  url: 'https://docs.renovate.com',
+  baseUrl: '/',
+  onBrokenLinks: 'warn',
+  onDuplicateRoutes: 'throw',
+  favicon: 'img/logo.png',
+  themeConfig: {
+    colorMode: {
+      disableSwitch: true,
+      respectPrefersColorScheme: true,
+    },
+    navbar: {
+      hideOnScroll: true,
+      title: 'Renovate Docs',
+      logo: {
+        alt: 'Renovate Logo',
+        src: 'img/logo.png',
+      },
+      items: [
+        {
+          href: 'https://github.com/renovatebot/renovate',
+          label: 'GitHub',
+          position: 'right',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
+      links: [],
+      logo: {
+        alt: 'Renovate Logo',
+        src: '/img/logo.png',
+      },
+      copyright: `Built with Daucusaurus v2`,
+    },
+    prism: {
+      theme,
+    },
+  },
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        debug: true,
+        docs: {
+          path: 'docs',
+          routeBasePath: '/',
+          sidebarPath: require.resolve('./sidebars.js'),
+          // editUrl: 'https://github.com/renovatebot/renovate/edit/master/docs/',
```

Comment:
```suggestion
          editUrl: 'https://github.com/renovatebot/renovate/edit/master/docs/',
```

I really like the `editUrl` feature from Docusuarus v2, it makes it really easy to fix a small typo or other small problem right away, without having to search trought the code for the exact file to fix. So I would recommend turning this on.

Is there a reason you have disabled this?

Reply:
we move the files before generating the docs, so the links would be wrong. we can fix that after switching to docusaurus

Reply:
Ah okay, I do hope we can turn it on soon after migrating! 😄